-code-checks

Logs: https://cmssdt.cern.ch/SDT/code-checks/cms-sw-PR-32971/21209

ERROR: Build errors found during clang-tidy run.
```
RecoPPS/Local/interface/RPixClusterToHit.h:13:10: error: 'CondFormats/PPSObjects/interface/PPSPixelTopology.h' file not found [clang-diagnostic-error]
#include "CondFormats/PPSObjects/interface/PPSPixelTopology.h"
         ^
Suppressed 2326 warnings (2325 in non-user code, 1 with check filters).
--
gmake: *** [config/SCRAM/GMake/Makefile.coderules:128: code-checks] Error 2
gmake: *** [There are compilation/build errors. Please see the detail log above.] Error 2
```

--- a/RecoPPS/Local/plugins/CTPPSPixelClusterProducer.cc
+++ b/RecoPPS/Local/plugins/CTPPSPixelClusterProducer.cc
@@ -3,8 +3,7 @@
 #include "RecoPPS/Local/interface/CTPPSPixelClusterProducer.h"
 
 CTPPSPixelClusterProducer::CTPPSPixelClusterProducer(const edm::ParameterSet &conf)
-    : tokenCTPPSPixelDigi_(
-          consumes<edm::DetSetVector<CTPPSPixelDigi> >(conf.getParameter<edm::InputTag>("label"))),
+    : tokenCTPPSPixelDigi_(consumes<edm::DetSetVector<CTPPSPixelDigi> >(conf.getParameter<edm::InputTag>("label"))),
       tokenCTPPSPixelAnalysisMask_(esConsumes()),
       tokenGainCalib_(esConsumes()),
       verbosity_(conf.getUntrackedParameter<int>("RPixVerbosity")),

--- a/RecoPPS/Local/plugins/CTPPSPixelClusterProducer.cc
+++ b/RecoPPS/Local/plugins/CTPPSPixelClusterProducer.cc
@@ -3,7 +3,7 @@
 #include "RecoPPS/Local/interface/CTPPSPixelClusterProducer.h"
 
 CTPPSPixelClusterProducer::CTPPSPixelClusterProducer(const edm::ParameterSet &conf)
-    : tokenCTPPSPixelDigi_(consumes<edm::DetSetVector<CTPPSPixelDigi> >(conf.getParameter<edm::InputTag>("label"))),
+    : tokenCTPPSPixelDigi_(consumes<edm::DetSetVector<CTPPSPixelDigi> >(conf.getParameter<edm::InputTag>("tag"))),
       tokenCTPPSPixelAnalysisMask_(esConsumes()),
       tokenGainCalib_(esConsumes()),
       verbosity_(conf.getUntrackedParameter<int>("RPixVerbosity")),
@@ -16,7 +16,7 @@ CTPPSPixelClusterProducer::~CTPPSPixelClusterProducer() {}
 void CTPPSPixelClusterProducer::fillDescriptions(edm::ConfigurationDescriptions &descriptions) {
   edm::ParameterSetDescription desc;
   desc.addUntracked<int>("RPixVerbosity", 0);
-  desc.add<edm::InputTag>("label", edm::InputTag("ctppsPixelDigis"));
+  desc.add<edm::InputTag>("tag", edm::InputTag("ctppsPixelDigis"));
   desc.add<int>("SeedADCThreshold", 2);
   desc.add<int>("ADCThreshold", 2);
   desc.add<double>("ElectronADCGain", 135.0);

--- a/RecoPPS/Local/plugins/CTPPSPixelLocalTrackProducer.cc
+++ b/RecoPPS/Local/plugins/CTPPSPixelLocalTrackProducer.cc
@@ -70,7 +70,7 @@ private:
 //------------------------------------------------------------------------------------------------//
 
 CTPPSPixelLocalTrackProducer::CTPPSPixelLocalTrackProducer(const edm::ParameterSet &parameterSet) {
-  inputTag_ = parameterSet.getParameter<edm::InputTag>("label");
+  inputTag_ = parameterSet.getParameter<edm::InputTag>("tag");
   verbosity_ = parameterSet.getUntrackedParameter<int>("verbosity");
   maxHitPerRomanPot_ = parameterSet.getParameter<int>("maxHitPerRomanPot");
   maxHitPerPlane_ = parameterSet.getParameter<int>("maxHitPerPlane");
@@ -119,8 +119,8 @@ CTPPSPixelLocalTrackProducer::~CTPPSPixelLocalTrackProducer() {}
 void CTPPSPixelLocalTrackProducer::fillDescriptions(edm::ConfigurationDescriptions &descriptions) {
   edm::ParameterSetDescription desc;
 
-  desc.add<edm::InputTag>("label", edm::InputTag("ctppsPixelRecHits"))
-      ->setComment("label of the RecHits input for the tracking algorithm");
+  desc.add<edm::InputTag>("tag", edm::InputTag("ctppsPixelRecHits"))
+      ->setComment("inputTag of the RecHits input for the tracking algorithm");
   desc.add<std::string>("patternFinderAlgorithm", "RPixRoadFinder")->setComment("algorithm type for pattern finder");
   desc.add<std::string>("trackFinderAlgorithm", "RPixPlaneCombinatoryTracking")
       ->setComment("algorithm type for track finder");

